src/main/java/uk/gov/hmcts/reform/et/syaapi/helper/TseApplicationHelper.java

```diff
@@ -98,6 +104,8 @@ public static void setRespondentApplicationWithResponse(RespondToApplicationRequ
         TseRespondType responseToAdd = request.getResponse();
         responseToAdd.setDate(TseApplicationHelper.formatCurrentDate(LocalDate.now()));
         responseToAdd.setFrom(CLAIMANT);
+        responseToAdd.setDateTime(TseApplicationHelper.getCurrentDateTime());
```

Comment:
Method already exists within the class so don't need to reference the class. 

```suggestion
        responseToAdd.setDateTime(getCurrentDateTime());
```

Alternatively replace the method and just call it directly

```suggestion
        responseToAdd.setDateTime(LocalDateTime.now().format(OLD_DATE_TIME_PATTERN));
```

Reply:
I went with the first solution in case the functionality is needed in the future and consistency with similar functions on the page